common/persistence/src/main/java/app/coronawarn/server/common/persistence/domain/normalization/NormalizableFields.java

```diff
@@ -0,0 +1,50 @@
+/*-
+ * ---license-start
+ * Corona-Warn-App
+ * ---
+ * Copyright (C) 2020 SAP SE and all other contributors
+ * ---
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ---license-end
+ */
+
+package app.coronawarn.server.common.persistence.domain.normalization;
+
+import app.coronawarn.server.common.persistence.domain.DiagnosisKey;
+
+/**
+ * This data structure is just a container for all fields of a
+ * {@link DiagnosisKey} to which normalization can be applied.
+ */
+public final class NormalizableFields {
+
+  private final int transmissionRiskLevel;
+  private final int daysSinceOnsetOfSymptoms;
```

Comment:
if we would use `Integer` objects instead, it's easier to also have mappings available for `0`, because we can check on `null` object

Reply:
I agree it would be good, but the problem is the TEK data structure does not have Integer as data types for its fields. This means that we wouldn't know if the mobile sent 0 or did not send an int field at all.
See the generated protobuf specification class TemporaryExposureKey

Reply:
Nevermind the previous answer. After checking again, we can know if a field was set in the protobuf or not. 
I will make some changes to the missing field checks which you can review again